### PR TITLE
Fix sensor framework timer issue and update sensor sample

### PR DESCRIPTION
**Pull request by wenyongh**
Fix the sensor framework timer issue reported by #884 when setting
`ms_to_next_check`, and modify the type of time related args/vars to
uint32 and fix the compilation warnings.

And update the sensor sample by creating two sensors to confirm that
the fix works correctly.

### CODE REVIEW

File: core/app-framework/sensor/native/runtime_sensor.c [hunk 1]
```diff
@@ -395,12 +395,12 @@ check_sensor_timers()
...
             s->last_read = now;
 
-            if (ms_to_next_check == -1 || (ms_to_next_check < s->read_interval))
+            if (s->read_interval < ms_to_next_check)
                 ms_to_next_check = s->read_interval;
         }
         else {
-            int remaining = s->read_interval - elpased_ms;
-            if (ms_to_next_check == -1 || (ms_to_next_check < remaining))
+            uint32 remaining = s->read_interval - elpased_ms;
```
Comment (lum1n0us) [inline core/app-framework/sensor/native/runtime_sensor.c:402]: shall we be worried if `s->read_interval` is smaller than `elpased_ms` ?

Comment (wenyongh) [reply core/app-framework/sensor/native/runtime_sensor.c:402]: This is the else branch, s->read_interval <= elpased_ms is handled in if branch, see Line 385:
`if (elpased_ms >= s->read_interval) {`

**Review by lum1n0us — APPROVED** [review]
LGTM

File: samples/simple/wasm-apps/sensor.c [hunk 2]
```diff
@@ -6,46 +6,76 @@
...
-        sensor_config(sensor, 0, 0, 0);
+    if (NULL != sensor1) {
+        sensor_config(sensor1, 0, 0, 0);
     }
+
+    if (NULL != sensor2) {
+        sensor_config(sensor2, 0, 0, 0);
+    }
+
+    free(user_data);
```
Comment (xujuntwt95329) [inline samples/simple/wasm-apps/sensor.c:77]: Had better check if `user_data` is NULL

Comment (wenyongh) [reply samples/simple/wasm-apps/sensor.c]: Done

**Pull request merged**